indico/modules/events/registration/forms.py

```diff
@@ -423,3 +423,24 @@ class RejectRegistrantsForm(IndicoForm):
 
     def is_submitted(self):
         return super().is_submitted() and 'submitted' in request.form
+
+
+class RegistrationTagForm(IndicoForm):
+    """
+    Form to create a new registration tag.
+    """
```

Comment:
```suggestion
    """Form to create/update a registration tag."""
```

same in the next class; these docstrings fit nicely in one line